client/src/pages/AssignRoles.js

```diff
@@ -32,13 +36,95 @@ const useStyles = makeStyles({
     button: {
         align: 'center',
         width: '120px'
+    },
+    leaveButton: {
+      marginTop: '20px',
+      align: 'center',
+      width: '120px',
+      backgroundColor: '#f23f3f',
+      color: 'white',
+    },
+    stayButton: {
+      marginTop: '30px',
+      align: 'center',
+      width: '120px',
+      color: 'black',
+    },
+    leaveButtonInner: {
+      marginLeft: '40px',
+      marginTop: '30px',
+      align: 'center',
+      width: '120px',
+      backgroundColor: '#f23f3f',
+      color: 'white',
+    },
+    modal : {
+      textAlign: 'center',
+      paddingTop: '20px',
+    },
+    popupHeader: {
+      fontSize: "20px",
+      color: 'red',
     }
 });
 
 export default function AssignRoles() {
 
     const { matchState } = useContext(MatchContext);
     const classes = useStyles();
+
+    const history = useHistory();
+
+    const askForLeave = async () => {
+      fetch(`/api/match/delete/${matchState.matchId}`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        }})
+        .then((responce) => {
+          console.log(responce);
+        })
+        .catch((err) => {
+          console.log(err);
+        });
```

Comment:
are we going to do anything with these responses?

Reply:
No, there is no further uses for response, removed this